webapp/src/js/services/recurring-process-manager.js

```diff
@@ -23,12 +32,20 @@ angular.module('inboxServices').factory('RecurringProcessManager',
           false //don't digest
         );
       },
-      stopUpdateRelativeDate: function() {
-        if (recurringProcesses.updateRelativeDates) {
-          $interval.cancel(recurringProcesses.updateRelativeDates);
-          recurringProcesses.updateRelativeDates = false;
+      stopUpdateRelativeDate: () => stopRecurringProcess('updateRelativeDates'),
+      startUpdateReadDocsCount: () => {
+        if (recurringProcesses.updateReadDocsCount) {
+          $interval.cancel(recurringProcesses.updateReadDocsCount);
```

Comment:
It's not immediately obvious why we'd cancel and then re-create the `updateReadDocsCount` interval here (as opposed to exiting the function early). This may be just me, but consider a comment here.

Reply:
Done (switched to returning early).